capactio/capact
# Move TypeInstance relations definition to Implementation.spec

**Description**

Changes proposed in this pull request:

- Move TypeInstance relations definition to `Implementation.spec`
- Make `outputTypeInstanceRelations` property required
- Update Content Creation and TypeInstance proposal docs

## Testing
- Run
```bash
ENABLE_POPULATOR=false make dev-cluster
```

- Populate data:
```bash
kubectl -n neo4j port-forward svc/neo4j-neo4j 7687:7687

APP_JSONPUBLISHADDR=http://{ip} APP_LOGGER_DEV_MODE=true go run cmd/populator/main.go .
```

- Navigate to https://gateway.voltron.local.
- Run query `Implementations` from https://github.com/Project-Voltron/go-voltron/blob/impl-output-typeinstance-rel/och-js/graphql/public/examples.graphql
- Observe new `outputTypeInstanceRelations` property
